src/update-data-feeds.ts

```diff
@@ -119,6 +118,15 @@ export const initializeUpdateCycle = async (
 
   logger.debug(`Initializing updates`, logOptions);
 
+  if (
+    (dataFeedType === DataFeedType.Beacon && beacons.length === 0) ||
+    (dataFeedType === DataFeedType.BeaconSet && beaconSets.length === 0)
```

Comment:
Nit: since we are already importing lodash `_.isEmpty()` on this file we might as well use it:

```suggestion
    (dataFeedType === DataFeedType.Beacon && isEmpty(beacons)) ||
    (dataFeedType === DataFeedType.BeaconSet && isEmpty(beaconSets))
```